Kitodo/src/test/java/org/kitodo/selenium/testframework/pages/EditPage.java

```diff
@@ -35,6 +39,15 @@ void addRow(List<WebElement> tableRows, String title, WebElement dialog) {
         for (WebElement tableRow : tableRows) {
             if (Browser.getCellDataByRow(tableRow, 0).equals(title)) {
                 clickLinkOfTableRow(tableRow);
+
+                // Hold up for some seconds…
+                try {
+                    Thread.sleep(3 * 1000);
+                } catch (InterruptedException e) {
+                    Logger logger = LogManager.getLogger(Browser.class);
```

Comment:
Logger class name is wrong: It should be `EditPage.class` instead of `Browser.class`. Maybe you can create the logger instance like on all other places in the code and not introduce a new variant.

Reply:
> Maybe you can create the logger instance like on all other places in the code and not introduce a new variant.

That's an unnecessary snarky comment.